website/src/components/Sponsor.tsx

```diff
@@ -30,7 +31,7 @@ export const OpenCollective: React.FC<{}> = () => (
           <h3>Sponsors</h3>
           <p className="p--desc ">
             Become a sponsor and get your logo on our README on GitHub with a
-            link to your site.
+            link to your site. 
```

Comment:
Added this one on purpose:
![image](https://github.com/react-native-elements/react-native-elements/assets/1556356/51b5ba39-dec2-4176-a819-a225b912cedb)